src/sentry/web/urls.py

```diff
@@ -200,7 +200,7 @@
                     name="sentry-login",
                 ),
                 url(
-                    r"^login/(?P<organization_slug>[^/]+)/$",
+                    r"^login/(?P<organization_slug_or_id>[^/]+)/$",
                     AuthOrganizationLoginView.as_view(),
```

Comment:
We could have another url router like `r"^login/id/(?P<organization_id>[^/]+)/$"` that points to `AuthOrganizationIdentifierLoginView`. This view class could handle translating `organization_id` to `organization_slug`, and the forwards requests to `AuthOrganizationLoginView.as_view()`.